application/helpers/members.php

```diff
@@ -128,4 +128,4 @@ public function gravatar($email, $s = 80, $d = 'mm', $r = 'g', $img = FALSE, $at
 		}
 		return $url;
 	}
-}
\ No newline at end of file
+}
```

Comment:
I am assuming there was a syntax error(missing bracket) which was added to fix the issue